### ISSUE TITLE
check subfontindex numbering

### ISSUE THREAD

**Issue by u-fischer** [u-fischer/luaotfload#86]
see comment from @adamliter here https://github.com/lualatex/luaotfload/issues/392#issuecomment-378097189

**Comment by adamliter** [comment]
@u-fischer I never ugpraded from MacTeX 2018 to MacTeX 2019, but now with MacTeX 2020, I no longer seem to have the same off-by-one issue. However, I do not seem to be able to set bold and italic fonts via the font index with LuaTeX. I just asked about this on [TeX.SX](https://tex.stackexchange.com/q/561172/32888), though perhaps it should have been a bug report? I'm not sure what the issue is ... It very well could be user error on my part. 🙃

**Comment by adamliter** [comment]
(cc @wspr )

**Comment by zauguin** [comment]
@adamliter Yes, the off-by-one issue has been fixed in most situations. (This is mostly still open because I'm not sure if we caught all of them)

The StackExchange question is a fontspec thing, see my TeX.sx answer for details. (It could be seen as a fontspec bug, so you might want to report it at the [fontspec issue tracker](https://github.com/wspr/fontspec/issues))

**Comment by adamliter** [comment]
@zauguin Thanks for the help. I really appreciate it! I've created a report at https://github.com/wspr/fontspec/issues/431.
